README.rst

```diff
@@ -139,6 +139,8 @@ Compatibility Matrix
 +-----------------------+---------------------------+
 | **BigchainDB Server** | **BigchainDB Driver**     |
 +=======================+===========================+
+| ``>= 2.0.0b1``        | ``0.5.2``                 |
```

Comment:
shouldn't the BigchainDB Server column say `>= 2.0.0b5` here?

Reply:
Yes, good catch.